app/models/bot/smooch.rb

```diff
@@ -696,12 +696,37 @@ def self.send_error_message(message, is_supported)
   def self.send_message_to_user(uid, text, extra = {}, force = false, preview_url = true)
     return if self.config['smooch_disabled'] && !force
     if RequestStore.store[:smooch_bot_provider] == 'TURN'
-      self.turnio_send_message_to_user(uid, text, extra, force, preview_url)
+      response = self.turnio_send_message_to_user(uid, text, extra, force, preview_url)
     elsif RequestStore.store[:smooch_bot_provider] == 'CAPI'
-      self.capi_send_message_to_user(uid, text, extra, force, preview_url)
+      response = self.capi_send_message_to_user(uid, text, extra, force, preview_url)
     else
-      self.zendesk_send_message_to_user(uid, text, extra, force, preview_url)
+      response = self.zendesk_send_message_to_user(uid, text, extra, force, preview_url)
     end
+    # store a TiplineMessage
+    external_id = self.get_id_from_send_response(response)
+    sent_at = DateTime.now
+    payload_json = { text: text }.merge(extra).to_json
+    team_id = self.config['team_id'].to_i
+    platform = RequestStore.store[:smooch_bot_platform] || 'unknown'
```

Comment:
Sawy, I think we are using "Unknown" (with capital "U") as the fallback... please confirm and change.

Reply:
Changed to `Unknown`